ml_git/relationship/models/spec_version.py

```diff
@@ -0,0 +1,77 @@
+"""
+© Copyright 2021 HP Development Company, L.P.
+SPDX-License-Identifier: GPL-2.0-only
+"""
+import json
+
+from ml_git.constants import DATASET_SPEC_KEY, LABELS_SPEC_KEY, MODEL_SPEC_KEY, STORAGE_SPEC_KEY, V1_STORAGE_KEY
+from ml_git.relationship.models.storage import Storage
+
+
+class SpecVersion:
+    """Class that represents an ml-entity spec version.
+
+    Attributes:
+        name (str): The name of the entity.
+        entity_type (str): The type of the ml-entity (datasets, models, labels).
+        version (str): The version of the ml-entity.
+        tag (str): The tag of the ml-entity spec version.
+        mutability (str): The mutability of the ml-entity.
+        categories (list): Labels to categorize the entity.
+        storage (Storage): The storage of the ml-entity.
+        amount (str): The amount of the version files.
```

Comment:
```suggestion
        amount (str): The amount of the versioned files.
```
I was thinking about we use `total_files`, `versioned_files`, or `total_versioned_files`... I believe amount is to generic...

Reply:
I agree. The attribute was renamed to `total_versioned_files`. 
This change was made in 06ec778.